src/beamlime/config/instruments/bifrost.py

```diff
@@ -144,3 +171,29 @@ def _all() -> StreamProcessor:
 f144_attribute_registry = {
     'detector_rotation': {'units': 'deg'},
 }
+
+
+def _make_bifrost_detectors() -> StreamLUT:
+    """
+    Bifrost detector mapping.
+
+    Input keys based on
+    https://confluence.ess.eu/display/ECDC/Kafka+Topics+Overview+for+Instruments
+    """
+    # Source names have the format `arc=[0-4];triplet=[0-8]`.
+    return {
+        InputStreamKey(
+            topic='bifrost_detector', source_name=f'arc={arc};triplet={triplet}'
+        ): f'arc{arc}_triplet{triplet}'
+        for arc in range(5)
+        for triplet in range(9)
+    }
+
+
+stream_mapping_dev = make_dev_stream_mapping(
+    'bifrost', detectors=list(detectors_config['fakes'])
+)
+stream_mapping = StreamMapping(
+    **make_common_stream_mapping_inputs(instrument='bifrost'),
+    detectors=_make_bifrost_detectors(),
+)
```

Comment:
Just a subjective nit pick, but maybe it would be nice to have something like 
```suggestion
stream_mapping['default'] = make_stream_mapping(instrument='bifrost',  detectors=_make_bifrost_detectors, env='default')
stream_mapping['dev'] = make_stream_mapping(instrument='bifrost',  detectors=detectors_config['fakes'], env='dev')
```
It would be easier to add more keys if we want to run in multiple different envs, and makes it look consistent.

It's fine either way!